torchaudio/compliance/kaldi.py

```diff
@@ -4,11 +4,19 @@
 
 
 __all__ = [
-    'spectrogram'
+    'fbank',
+    'get_mel_banks',
+    'inverse_mel_scale',
+    'inverse_mel_scale_scalar',
+    'mel_scale',
+    'mel_scale_scalar',
+    'spectrogram',
+    'vtln_warp_freq',
+    'vtln_warp_mel_freq',
 ]
 
-# numeric_limits<float>::epsilon()
-EPSILON = torch.tensor(1.19209290e-07, dtype=torch.get_default_dtype())
+# numeric_limits<float>::epsilon() 1.1920928955078125e-07
+EPSILON = torch.tensor(torch.finfo(torch.float).eps, dtype=torch.get_default_dtype())
```

Comment:
Did you compare to Kaldi when running this using doubles? If not, it might make sense to throw an error in our compliance functions if the users passes in a Tensor of doubles, or add some documentation that sets the expectation that this only works with doubles (which is enough anyway).